content-src/components/DiscoveryStreamComponents/DSLinkMenu/_DSLinkMenu.scss

```diff
@@ -0,0 +1,21 @@
+.ds-hero-item,
+.ds-list-item,
+.ds-card {
+  @include context-menu-button;
+  // For max-width  1378px, with $break-point-widest 1042 px
+  // and $card-width 224px
+  @media (max-width: $break-point-widest + $card-width * 1.5) {
+    &.last-item {
```

Comment:
@Mardak This PR handles last item show in card grid and lists to show context-menu to the left. However this doesn't work for complex layout where we display card and list items in 2/3 and 1/3 width. This issue exists for Top Sites also when shown in 2/3 and 1/3 width when seen on dev-test-all